en/SUMMARY.md

```diff
@@ -414,3 +412,6 @@
   * [Translation](contribute/translation.md)
   * [Terminology/Notation](contribute/notation.md)
   * [Licenses](contribute/licenses.md)
+* [Releases](releases/README.md)
```

Comment:
FYI only pushed to end. Certainly shouldn't be "in the middle".

Reply:
Yeah the middle was weird, i think its best if we keep it towards the end